source/catalog.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>Седона</title>
+</head>
+
+<body class="page-body">
+  <header class="main-header">
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-image" src="/source/img/logo.svg" width="185" height="96" alt="Логотип Sedona">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a class="site-list__link" href="index.html">
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              Фото и видео
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              Форма Отзыва
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              HTML ACADEMY
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="welcome-block">
+      <h1 class="visually-hidden">седону</h1>
+      <img class="sedona-image" src="#" width="0" height="0" alt="#">
+    </section>
+    <section class="header-foto-video">
+      <h2 class="header-foto-video__title">Фото и видео</h2>
+      <p class="header-foto-video__text">Не можете решиться на путешествие из-за курса? Фотографии помогут вам забыть о
+        политике и экономике</p>
+    </section>
+    <section class="foto">
+      <ul class="foto__list">
+        <li class="foto__item">
+          <img class="foto__image" src="#" width="0" height="0" alt="Название фото">
+          <div class="foto__wrapper discription-foto">
```

Comment:
Надо грамматику английского подкачать бы

```suggestion
          <div class="foto__wrapper description-photo">
```